packages/traefik-dynamic/templates/traefik.toml

```diff
@@ -0,0 +1,52 @@
+# global config
+port = "{{or (key "config/traefik/port") ":80"}}"
```

Comment:
Github also thinks this is *just* toml, when it's toml+go template. It's a valid template which renders valid toml.